src/Hero.java

```diff
@@ -1,85 +1,10 @@
 import java.util.Random;
 
-public class Hero {
-    Random generator = new Random();
-    public Coordinates position = new Coordinates(0, 0);
-    char symbol;
+public class Hero extends Token{
```

Comment:
Token to niefortunne słowo - może zostać ale ma w programowaniu znaczenie (jwt token, auth token).